mxd/modules/connector/variables.tf

```diff
@@ -62,4 +62,21 @@ variable "ssi-config" {
     oauth-clientsecret = ""
     oauth-secretalias  = ""
   }
-}
\ No newline at end of file
+}
+
+variable "azure-account-name" {
```

Comment:
could provide a default value, something like `"${var.humanreadablename}-account` or similar

Reply:
Terraform doesn't support variable reference inside `variables.tf`. `terraform init` fails.
https://stackoverflow.com/questions/58841060/terraform-variables-within-variables

We do have default values for alice / bob accounts in top level variables.tf.

Reply:
ah right, forgot.